htdocs/core/class/commonobject.class.php

```diff
@@ -1061,18 +1061,25 @@ public function liste_contact($statut = -1, $source = 'external', $list = 0, $co
 	{
         // phpcs:enable
 		global $langs;
+        
+        $langs->load("dict");
 
 		$tab=array();
 
 		$sql = "SELECT ec.rowid, ec.statut as statuslink, ec.fk_socpeople as id, ec.fk_c_type_contact";    // This field contains id of llx_socpeople or id of llx_user
 		if ($source == 'internal') $sql.=", '-1' as socid, t.statut as statuscontact, t.login, t.photo";
 		if ($source == 'external' || $source == 'thirdparty') $sql.=", t.fk_soc as socid, t.statut as statuscontact";
```

Comment:
```suggestion
		if ($source == 'external' || $source == 'thirdparty') $sql.=", t.fk_soc as socid, t.statut as statuscontact, t.phone, t.phone_perso, t.phone_mobile, t.fax, t.email";
```

See the comment below :wink: